cli/src/commands/operation/diff.rs

```diff
@@ -384,14 +385,14 @@ fn write_modified_change_summary(
 ) -> Result<(), std::io::Error> {
     writeln!(formatter, "Change {}", short_change_hash(change_id))?;
     for commit in modified_change.added_commits.iter() {
-        formatter.with_label("diff", |formatter| write!(formatter.labeled("added"), "+"))?;
+        formatter.with_label("diff", |formatter| write_labeled!(formatter, "added", "+"))?;
```

Comment:
Interesting one being nested. Could make the macro accept something like
```suggestion
        write_labeled!(formatter, "diff" & "added", "+")?;
```

Reply:
label string could be separated by whitespace (and split by `formatter.labeled()`), but I don't know if we like it.